docs/user_guides/fs/vector_similarity_search.md

```diff
@@ -0,0 +1,96 @@
+---
+description: Users guide about how to use Hopsworks for vector similarity search 
+---
```

Comment:
Also please add an entry to this guide on this page: https://docs.hopsworks.ai/3.7/user_guides/fs/

Reply:
Should we not have links to latest?
https://docs.hopsworks.ai/latest/user_guides/fs/